eos-autoupdater/docs/eos-autoupdater.8

```diff
@@ -1,5 +1,6 @@
 .\" Manpage for eos-autoupdater.
-.\" Documentation is under the same licence as the eos-updater package.
+.\" SPDX-License-Identifier: CC-BY-SA-4.0
```

Comment:
```
git shortlog -se [all manpages touched by this patch]
```

reveals that all contributions have been made by Endless employees, except 2b732527fb316c3e9c386ccdad763421f6be2cae which was done under contract. So EOSF owns these files.

@ramcq Please approve this relicensing!

Reply:
Approved!